src/nopemd.h.in

```diff
@@ -348,6 +348,20 @@ NMDAPI struct nmd_frame *nmd_get_next_frame(struct nmd_ctx *s);
 /* Release a frame obtained with nmd_get_frame() */
 NMDAPI void nmd_frame_releasep(struct nmd_frame **framep);
 
+/**
+ * Render to surface and release a MediaCodec frame obtained with
+ * nmd_get_frame().
```

Comment:
could be obtained by other means though (ms version or get next frame one)

Reply:
Done. Also updated nmd_frame_releasep() documentation in a separate commit.